src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -1728,6 +1739,17 @@ public boolean saveTown(Town town) {
 		// Outlaws
 		list.add("outlaws=" + StringMgmt.join(town.getOutlaws(), ","));
 
+		// Metadata
+		StringBuilder md = new StringBuilder();
+		if (town.hasMeta()) {
+			HashSet<CustomDataField> tdata = town.getMetadata();
+			for (CustomDataField cdf : tdata) {
+				md.append(cdf.toString()).append(";");
+			}
+		}
+		Towny.getPlugin().getLogger().info(md.toString());
```

Comment:
Either nuke this logger bit or switch it over to use TownyMessaging.sendDebugMessage()